npm/install.js

```diff
@@ -0,0 +1,82 @@
+const fs = require("fs");
+const https = require("https");
+const path = require("path");
+
+const packageJson = require("./package.json");
+const { version, name, repository } = packageJson;
+
+async function install() {
+  const downloadURL = getDownloadURL();
+
+  try {
+    const binDir = path.join(__dirname, "bin");
```

Comment:
What is `__dirname` in this context? I'm guessing it will be the context of the global `npm i -g` command we instruct users to execute?

Reply:
The absolute path of the directory where NPM has installed our package.